### PR TITLE
New case study + new privacy policy & terms of service

### PR DESCRIPTION
**Pull request by yashghelani**

### CODE REVIEW

File: app/routes/blog/introducing-technical-blogs.mdx [hunk 1]
```diff
@@ -73,11 +73,11 @@ Our value proposition to engineering teams:
 - Turnkey data plumbing for your lake or warehouse ($1M in annual cost savings)
 - Easy like SaaS but works in your cloud prem - sign up and go!
 - Privacy by design, **so engineers don’t have to be lawyers**
-- [Open Sourcing](https://github.com/blotoutio) the common core -- if we die you keep it
+- [Open Sourcing](https://github.com/blotoutio) the common core—if we die you keep it
 
 ## What are we not?
 
-Blotout is not a data pipe or workflow company. We provide intelligence as data infrastructure -- whether it’s risk, ROI, or customer journeys -- in your cloud premise. We further enable comprehensive insights across the span and breadth of your data. Finally, we understand that data is a multi-organizational priority, so we help you break the silos.
+Blotout is not a data pipe or workflow company. We provide intelligence as data infrastructure—whether it’s risk, ROI, or customer journeys—in your cloud premise. We further enable comprehensive insights across the span and breadth of your data. Finally, we understand that data is a multi-organizational priority, so we help you break the silos.
```
Comment (NejcZdovc) [inline app/routes/blog/introducing-technical-blogs.mdx:80]: is this correct? `infrastructure—whether` doesn't make sense

**Review by NejcZdovc — COMMENTED** [review]
as far as I can see we should be replacing `--` with `-`

File: app/routes/blog/introducing-technical-blogs.mdx (hunk 1 above)
Comment (yashghelani) [reply app/routes/blog/introducing-technical-blogs.mdx:80]: It’s actually an em dash and not a hyphen. In a monospace font, it’s difficult to differentiate between that and a hyphen. This is how it looks -

Screenshot 2021-12-01 at 10 43 17 AM
s

Comment (yashghelani) [reply app/routes/blog/introducing-technical-blogs.mdx:80]: We’re supposed to replace double hyphens with an em dash and not a single hyphen. Em dashes are better suited for such situations as the sentence in between the dashes is connected to both the words before and after.

**Review by NejcZdovc — APPROVED** [review]

**Pull request merged**